test/tests/cht/multi_cht/nek_pp.i

```diff
@@ -0,0 +1,268 @@
+[Mesh]
+  type = NekRSMesh
+  volume = true
+[]
+
+[Problem]
+  type = NekRSStandaloneProblem
+  casename = 'conj_ht'
+  output = 'temperature pressure'
+[]
+
+# This AuxVariable and AuxKernel is only here to get the postprocessors
+# to evaluate correctly. This can be deleted after MOOSE issue #17534 is fixed.
```

Comment:
This is a side note not related to this PR, but does this MOOSE issue affect `NekRSSeparateDomainProblem` specifically for SAM-NekRS or THM-NekRS coupling?

Reply:
This MOOSE issue affects any class that derives from `ExternalProblem` (all the NekRS classes are affected, as well as the OpenMC ones).